apps/aecore/src/aec_chain.erl

```diff
@@ -103,7 +103,7 @@ get_account(PubKey) ->
     end.
 
 get_account_at_hash(PubKey, Hash) ->
-    case get_block_state(Hash) of
+    case get_block_state_partial(Hash, [accounts]) of
```

Comment:
Switch the trees to a dirty backend :)
Here when we walk the MPT we generally access every MPT node as a full DB transaction.